extensions/amp-tiktok/0.1/amp-tiktok.css

```diff
@@ -29,3 +29,13 @@
     width: 325px;
     height: 500px;
 }
+
+.i-amphtml-tiktok-placeholder-image {
+    height: 578px;
+    top: 1px;
+    width: 325px;
+    left: 50%;
+    transform: translateX(-50%);
+    position: absolute;
```

Comment:
This is the same as `.i-amphtml-tiktok-centered`. I would suggest you remove these props, and add `i-amphtml-tiktok-centered` to the element's classlist before `i-amphtml-tiktok-placeholder-image`.

```suggestion
```